MatthewVanEenenaam/Haduoken.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Haduoken : MonoBehaviour {
+
+    
+    private float speed = 7;
+    private Animator anim;
+    
+public Player player
+    {
+        get;
+        set;
+    }
+
+    public bool towardsPlayer
+    {
+        get;
+        set;
+    }
+
+
+    private void Start() {
+
+        anim = GetComponent<Animator>();
+        player = GameObject.FindGameObjectWithTag("PlayerScriptHolder").GetComponent<Player>();
+        towardsPlayer = true;
+    }
+
+
+    // Update is called once per frame
+    void Update() {
+
+        if (!towardsPlayer){
+            GetComponent<Rigidbody2D>().velocity = transform.up * speed;
+            anim.SetBool("GoingBack", true);
+        }
+        else{
+            GetComponent<Rigidbody2D>().velocity = transform.up * -speed;
+        }
+    }
+
+
+    public void OnTriggerEnter2D(Collider2D col){
```

Comment:
Klein dingetje, ben niet helemaal zeker maar vallen haakjes onder spaties tussen alle operators? Want het lijkt mij dat () { mooier is en dit is bij de Start en Update ook zo.

Reply:
deze haakjes regel is gonna be the death of me man shit hahaha. im on it.